glyphs2fontir/src/toir.rs

```diff
@@ -49,34 +49,27 @@ fn to_ir_path(glyph_name: GlyphName, src_path: &Path) -> Result<BezPath, WorkErr
     // See also https://github.com/fonttools/ufoLib2/blob/4d8a9600148b670b0840120658d9aab0b38a9465/src/ufoLib2/pointPens/glyphPointPen.py#L16
     let mut path_builder = GlyphPathBuilder::new(glyph_name.clone());
     if src_path.nodes.is_empty() {
-        return Ok(path_builder.build());
+        return Ok(path_builder.build()?);
     }
 
-    let mut nodes = &src_path.nodes[..];
+    let mut nodes: Vec<_> = src_path.nodes.iter().collect();
```

Comment:
This could just be `src_path.nodes.clone()`?

Reply:
Done. Why didn't clippy suggest that?!